src/main/java/net/sf/jabref/gui/EntryTypeList.java

```diff
@@ -127,16 +126,6 @@ public void addDefaultActionListener(ActionListener l) {
     }
 
     @Override
-    public void actionPerformed(ActionEvent e) {
-        // Default button pressed.
-        if (e.getSource() == def) {
-            def.setEnabled(false);
-        } else {
-            super.actionPerformed(e);
```

Comment:
Ok, thanks! It might have been unnecessary during all of its existence, or maybe it had a purpose at some point in time that got lost. In any case, its removal is positive.